deps/wazuh_testing/wazuh_testing/scripts/data_visualizations.py

```diff
@@ -21,20 +49,32 @@ def get_script_arguments():
     parser.add_argument('-c', '--columns', dest='columns', default=None,
                         help=f'Path to Json with Columns to Plot. Default {None}.')
     parser.add_argument('-u', '--unify', dest='unify', action='store_true',
-                        help=f'Unify data of the binary processes with their subprocesses to plot.')
+                        help='Unify data of the binary processes with their subprocesses to plot.')
 
     return parser.parse_args()
 
 
 def main():
     options = get_script_arguments()
-    destination = options.destination
+    create_destination_directory(options.destination)
+
+    target = options.visualization_target
+    validate_arguments(options)
+
+    if target in ['analysis', 'remote', 'wazuhdb']:
+        dv = DaemonStatisticsVisualizer(options.csv_list, daemon=target,
+                                        store_path=options.destination,
+                                        base_name=options.name)
+    elif target == 'binary':
+        dv = BinaryDatavisualizer(options.csv_list,
+                                    store_path=options.destination,
+                                    base_name=options.name,
+                                    unify_child_daemon_metrics=options.unify)
```

Comment:
Why don't we use the dictionary if `binary` is in `strategy_plot_by_target`? Maybe we could use `update` to avoid repetitive code.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5518/commits/0312c7e3d43a15190f56f9a75f2cfba1ad84c1bf
In addition, some minor docstring style errors and a minor refactor were performed:  https://github.com/wazuh/wazuh-qa/pull/5518/commits/be0afc8fab0a27f458b6a607d85797c31ac7a4fb, https://github.com/wazuh/wazuh-qa/pull/5518/commits/6c405dcc7fb97b261d4077134678f8561144524e, https://github.com/wazuh/wazuh-qa/pull/5518/commits/9430c86dec64776e1c6ef140b30ca8a555b42a0e